OpenRobertaServer/src/main/java/de/fhg/iais/roberta/persistence/dao/ProgramDao.java

```diff
@@ -253,9 +253,19 @@ public JSONArray loadGallery(int galleryId, int userId) {
             "where p.OWNER_ID = :galleryId " + //
             "  and p.ROBOT_ID = r.ID " + //
             "  and p.AUTHOR_ID = u.ID ";
+
+        if(!robotGroup.isEmpty()){
```

Comment:
Looks as if you haven't used our formatters. Please make sure you have installed the formatters like it is described in the wiki: https://github.com/OpenRoberta/openroberta-lab/wiki/Importing-into-Eclipse